scripts/process-trace.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env python3
```

Comment:
Please rename to `process_trace.py` so that this can be imported (you can't import a module whose name isn't a valid identifier, for example due to the hyphen).

Reply:
🚀